packages/protocol/migrations_sol/Migration.s.sol

```diff
@@ -296,50 +310,49 @@ contract Migration is Script, UsingRegistry, MigrationsConstants {
   }
 
   function migrateFeeCurrencyDirectory() public {
-    deployProxiedContract(
+    address feeCurrencyDirectoryProxyAddress = 0x9212Fb72ae65367A7c887eC4Ad9bE310BAC611BF;
```

Comment:
would move this to a parameter (afaik, the only way to do this with foundry is to generate a json file with the key/value pair). If something changes with the nonces of the txs in the script, then this address will change and thus the script break.